nunaliit2-multimedia/src/main/java/ca/carleton/gcrc/olkit/multimedia/ffmpeg/FFprobeProcessor.java

```diff
@@ -0,0 +1,54 @@
+package ca.carleton.gcrc.olkit.multimedia.ffmpeg;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import java.io.BufferedReader;
+import org.json.JSONObject;
+import org.json.JSONArray;
+import java.util.stream.Collectors;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class FFprobeProcessor {
+
+    private static final Logger log = LoggerFactory.getLogger(FFprobeProcessor.class);
+
+    static public String ffprobeCommand = "ffprobe -v quiet -print_format json -show_streams %s";

```

Comment:
Can probably make this public (or private) static final String + probably allcaps snake case the variable name